home/src/app/theme/components/searchbar/searchbarCtrl.js

```diff
@@ -28,6 +28,14 @@
 				$rootScope.mapPage = true;
 				$rootScope.mapSearchStringTrans = $scope.searchString;
 				$state.go('map', {}, { reload: true });
+			} else {
+				$location.path("/search");
+				if ($location.path() == "/search") {
+					searchService.prepForBroadcast($rootScope.predicates);
+					$scope.$on('handleBroadcast', function() {
+				        searchService.setBroadcast($rootScope.predicates);
```

Comment:
User will continues search good at search bar. It changes value $rootScope.predicates in searchService and executes search function in searchCtrl